src/System Application/App/Entity Text/src/EntityTextImpl.Codeunit.al

```diff
@@ -41,20 +51,26 @@ codeunit 2012 "Entity Text Impl."
 
     [NonDebuggable]
     procedure GenerateSuggestion(Facts: Dictionary of [Text, Text]; Tone: Enum "Entity Text Tone"; TextFormat: Enum "Entity Text Format"; TextEmphasis: Enum "Entity Text Emphasis"; CallerModuleInfo: ModuleInfo): Text
+    begin
+        exit(GenerateSuggestion(Facts, Tone, TextFormat, TextEmphasis, CallerModuleInfo, ''));
+    end;
+
+    [NonDebuggable]
+    procedure GenerateSuggestion(var Facts: Dictionary of [Text, Text]; var Tone: Enum "Entity Text Tone"; var TextFormat: Enum "Entity Text Format"; var TextEmphasis: Enum "Entity Text Emphasis"; var CallerModuleInfo: ModuleInfo; SourceScenarioName: Text): Text
     var
         SystemPrompt: Text;
         UserPrompt: Text;
```

Comment:
UserPrompt should be moved within the codeunit and made non-debuggable, so codeunit hides the internal value. Then we could have it as a global variable and not pass it around. This way we can have more code debuggable.